### PR TITLE
Fix: Show param type and return in reference page.

### LINKED ISSUE #562: The “Returns” section does not appear on the reference page for functions and methods that return values.
### Most appropriate sections of the p5.js website?

Reference

### What is your operating system?

None

### Web browser and version

_No response_

### Actual Behavior

The “Returns” section does not appear on the reference page for functions and methods that return values.

スクリーンショット 2024-09-21 20 42 40

### Expected Behavior

The “Returns” section does appear on the reference page for functions and methods that return values.

### Steps to reproduce

1. open https://p5js.org/reference/p5/cos/
2. scroll down to under Parameters.

e.g. function:
https://p5js.org/reference/p5/cos/
(old version: https://archive.p5js.org/reference/#/p5/cos )

e.g. method: 
https://p5js.org/reference/p5.Vector/copy/
(old version: https://archive.p5js.org/reference/#/p5.Vector/copy )

### Would you like to work on the issue?

I will create PR

### PR DESCRIPTION
**Pull request by shibomb**
Resolves #562 #564 
Sorry for multiple issues.

### Changes:
- for #564 : param type shows with simple param field.
- for #562 : show return info. and add UI label (en.yaml) for translations.

### CODE REVIEW

File: src/layouts/ReferenceItemLayout.astro [hunk 1]
```diff
@@ -169,6 +171,23 @@ const seenParams: Record<string, true> = {};
           </>
         )
       }
+      {
+        (entry.data.return) && (
+          <>
+            <div class="mb-xl">
+              <h2 class="mb-md text-h3">{t("Return")}</h2>
```
Comment (davepagurek) [inline src/layouts/ReferenceItemLayout.astro]: Looks like on the old site it was "Returns" with an s, maybe we can do the same here too?

image


```suggestion
              <h2 class="mb-md text-h3">{t("Returns")}</h2>
```

**Review by davepagurek — COMMENTED** [review]
Thanks, looks good! I just noticed one difference from before, but otherwise good to go.

File: src/layouts/ReferenceItemLayout.astro (hunk 1 above)
Comment (shibomb) [reply src/layouts/ReferenceItemLayout.astro]: @davepagurek 
Thanks for reply.

This is a suggestion for improvement.

Indeed, the old version was with "s".
I assume this “s” means plural, but can the returned value be plural?

- It is certainly possible to have multiple meanings by using arrays or Json, but I think that should be explained in the Description.

- As a property of the .mdx file is “return:”.

- Also, for example, there is no “s” in processing.org.
  (I haven't checked all the pages... For example, https://processing.org/reference/cos_.html .)

Please consider the above!

If you need the “s”, I will modify the necessary part :-)

Comment (davepagurek) [reply src/layouts/ReferenceItemLayout.astro]: Returns with an s is I think more common, e.g. [Tone.js](https://tonejs.github.io/docs/15.0.4/classes/Context.html#createWaveShaper), [React](https://react.dev/reference/react/useCallback#returns), [matter.js](https://brm.io/matter-js/docs/classes/Constraint.html), and [lodash](https://lodash.com/docs/4.17.15#now). I think the reason for it isn't to consider "return" as a plural noun, but rather as a verb, e.g. "this function returns a number", or just "returns: number" for short in this case. In this case the s just means present tense, "it returns" as opposed to "it returned" in the past.

Comment (shibomb) [reply src/layouts/ReferenceItemLayout.astro]: I understand.
I will correct it!

Comment (shibomb) [reply src/layouts/ReferenceItemLayout.astro]: @davepagurek 
I have updated the branch.
Please re-review.
Thank you.

**Review by davepagurek — APPROVED** [review]
Thanks, looks great!

**Pull request merged**